packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: 0.25;
```

Comment:
hmm.... on second thought. we would need to have some sort of fixed size on the input ... since `flex: 1` is applied to all inputs.